public/calendar_reboot.js

```diff
@@ -820,6 +872,27 @@ var LocalCalendarStorage = function(params) {
         return key;
     };
     
+
+    self.startWork = function(processName) {
+        // Will increment the counter and possibly fire an event.
+        console.log("Accessing storage from " + processName);
+        self.activeCalls += 1;
+        
+        // Will dispatch the event backgroundActivityChange 
+        if (self.activeCalls > 0) {
```

Comment:
Yeah, I understand. You mean if I had a customEvent for example I'd add activeCalls to it's details right? So, that you can access it. Except I'm not using customEvents anymore. 

So, it would be ... in the function call... that you are listening to?